packaging/Makefile

```diff
@@ -15,7 +15,7 @@ debian-package-code:
 	cp ../examples/instance_message_from_aleph.json ./aleph-vm/opt/aleph-vm/examples/instance_message_from_aleph.json
 	cp -r ../examples/data ./aleph-vm/opt/aleph-vm/examples/data
 	mkdir -p ./aleph-vm/opt/aleph-vm/examples/volumes
-	pip3 install --target ./aleph-vm/opt/aleph-vm/ 'aleph-message==0.4.1' 'jwskate==0.8.0' 'eth-account==0.9.0' 'sentry-sdk==1.31.0' 'qmp==1.1.0'
+	pip3 install --target ./aleph-vm/opt/aleph-vm/ 'aleph-message@git+https://github.com/aleph-im/aleph-message.git@mhh-pay-as-you-go-v1#egg=aleph-message' 'jwskate==0.8.0' 'eth-account==0.9.0' 'sentry-sdk==1.31.0' 'qmp==1.1.0' 'superfluid@git+https://github.com/Godspower-Eze/superfluid.py#egg=superfluid'
```

Comment:
It isn't, so I had to add as a Git repository directly

Reply:
That is annoying. We must at least pin the exact git commit we use in such cases, preferably also fork the repository and install from our fork or publish it ourserlves on PyPI. Motivation: We want to ensure it stays available and auditable in case of supply chain attack.